contracts/libraries/RMM01Lib.sol

```diff
@@ -100,19 +100,25 @@ library RMM01Lib {
             ? self.params.priorityFee
             : self.params.fee;
 
-        (data.virtualX, data.virtualY) =
-            self.getVirtualPoolReservesPerLiquidityInWad();
+        data.liquidity = self.liquidity;
+        (data.virtualX, data.virtualY) = self.getVirtualReservesWad();
```

Comment:
Uses the `getVirtualReservesWad` function which just returns the full virtual reserve amounts in WAD units.